charts/vh-video-web/values.yaml

```diff
@@ -86,4 +86,4 @@ java:
     VHSERVICES__ENABLEIOSTABLETSUPPORT: true
     VHSERVICES__ENABLEVIDEOFILTERS: true
     VHSERVICES__USERAPIURL: https://vh-user-api.{{ .Values.global.environment }}.platform.hmcts.net/
-    VHSERVICES__VIDEOAPIURL: https://vh-video-api.{{ .Values.global.environment }}.platform.hmcts.net/
+    VHSERVICES__VIDEOAPIURL: https://vh-video-api-pr-492.dev.platform.hmcts.net/
```

Comment:
need to set this back to what it was, this was just for testing

Reply:
leave this no change